components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-details/public/js/details.js

```diff
@@ -285,6 +294,62 @@ function subscriptionsTabHandler(event) {
 
 }
 
+/**
+ * Event handler for API access control detail tab onclick event;Get the current API access control tab HTML via UUFClient and display.
+ * @param event {object} Click event of the access control tab
+ */
+function accessControlTabHandler(event) {
+    var api_client = event.data.api_client;
+    var api_id = event.data.api_id;
+    api_client.get(api_id).then(
+        function (response) {
+            var api_data = JSON.parse(response.data);
+            var permission_data = api_data.permission;
+            var mode = "OVERWRITE"; // Available modes [OVERWRITE,APPEND, PREPEND]
+            var callbacks = {
+                onSuccess: function (data) {
+                    if (!permission_data || permission_data === "[]"){
+                        $('#no-roles-msg').removeClass('hide');
+                        $('#permissionTable').hide();
+                    }
+                },
+                onFailure: function (data) {
+                    console.debug("Failed");
```

Comment:
This will cause some issues on IE lower versions

Reply:
Thanks for the review. This file is no longer available in this repository.